source/index.html.md

```diff
@@ -218,6 +218,7 @@ error categories, the last two digits define specific errors.
 * 0707: 2FA Locked
 * 0708: Max attempts reached to perform operation
 * 0709: Action not permitted for missing KYC data
+* 0711: Verification completed. No missing requirements to fill
```

Comment:
is 0710 used for something else? Can't see it in the PHP codebase

Reply:
Yes, https://github.com/bitsoex/bitsoex/pull/3822 will be used here, whenever I can merge those PRs

Reply:
I saw @viccomx added the 0710 [here](https://github.com/bitsoex/slate/commit/1964a30fd137658da88367befd0d75038dd4baec) @thenewversion

Reply:
Of course